api/src/service/domain/workflow/subproject_projected_budget_delete.ts

```diff
@@ -62,8 +66,25 @@ export async function deleteProjectedBudget(
   if (isEqual(subproject.projectedBudgets, result.projectedBudgets)) {
     return { newEvents: [], projectedBudgets: result.projectedBudgets };
   } else {
+    // Create notification events:
+    const recipients = subproject.assignee
```

Comment:
Why is this differently implemented than in `api/src/service/domain/workflow/project_projected_budget_delete.ts`

Reply:
used same implementation for `project_projected_budget_delete`, `project_projected_budget_update`, `subproject_projected_budget_delete` and `subproject_projected_budget_update`